patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -129,3 +136,42 @@ function removeTrailingAndLeadingSlash (p: string) {
   }
   return p
 }
+
+async function filterIgnoredFiles (folder: string, includedFiles: string[]) {
+  const destinationDir = tempy.directory()
+  await fs.promises.mkdir(destinationDir, { recursive: true })
+
+  const files = await fs.promises.readdir(folder)
+  for (const file of files) {
```

Comment:
Maybe it would be better to use `packlist` to get the files that need to be copied, like here https://github.com/pnpm/pnpm/blob/main/fetching/directory-fetcher/src/index.ts#L129